tests/run_tests.py

```diff
@@ -767,6 +782,8 @@ def replace_global_values(gp, replace):
     gpl = []
     for line in iter(gp.splitlines()):
         line_list = line.split()
+        if line_list == []:
+            continue
```

Comment:
is this just skipping empty lines? If so, it would be cleaner to write it as:

```Python
if not line_list:
    continue
```

it may be better to go without the continue statement and just do:

```Python
if line_list:
    ... # complete the rest of the for loop
```